packages/openid4vc/src/openid4vc-verifier/OpenId4VcSiopVerifierService.ts

```diff
@@ -497,13 +524,13 @@ export class OpenId4VcSiopVerifierService {
     const builder = RP.builder()
 
     const responseTypes: ResponseType[] = []
-    if (!presentationDefinition && idToken === false) {
+    if (!(presentationDefinition && dcqlQuery) && idToken === false) {
       throw new CredoError('Either `presentationExchange` or `idToken` must be enabled')
```

Comment:
Message is not updated

Reply:
sorry. Not sure what do you mean?

Reply:
It says either presentationexchange or idtoken must be enabled.